pkg/s3select/csv/reader.go

```diff
@@ -294,6 +294,11 @@ func NewReader(readCloser io.ReadCloser, args *ReaderArgs) (*Reader, error) {
 		ret := csv.NewReader(r)
 		ret.Comma = []rune(args.FieldDelimiter)[0]
 		ret.Comment = []rune(args.CommentCharacter)[0]
+		ret.Quote = []rune{}
+		if len([]rune(args.QuoteCharacter)) > 0 {
+			// Add the first rune of args.QuoteChracter
+			ret.Quote = append(ret.Quote, []rune(args.QuoteCharacter)[0])
```

Comment:
Is this validated upstream to not have more than 1 rune?

Reply:
> Is this validated upstream to not have more than 1 rune?

Yes, Quote can be empty or contains only one character (utf8 character)